clientapp/src/utilities/dateTimeUtilites.js

```diff
@@ -0,0 +1,23 @@
+export const dateTime = {
+    getTimeZoneFormat: function(date){
+        const dateTimeFormat = new Intl.DateTimeFormat(
+            'en-US',
+            {
+                year: 'numeric',
+                month: '2-digit',
+                day: '2-digit',
+                hour: '2-digit',
+                minute: '2-digit',
+                second: '2-digit'
+            }
+        );
+        const [{ value: month },,{ value: day },,{ value: year },, {value: hour},,{value: minute},,{value:second}] = dateTimeFormat.formatToParts(date);
+
```

Comment:
i'm very confused by this syntax. do you know what this is doing?

Reply:
I have updated these lines to make it clear.